### PR TITLE
feat(api): finish license service v1; add pkpass support

### PR DESCRIPTION
**Pull request by Valur**
# Finish license service v1

Finishes v1 implementation of a generic license service, with driving license as the first (and only) implementation.

See more:
https://www.notion.so/License-service-WIP-d8a9df75b7d747d1950b7c155507ab7c

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by pshomov — APPROVED** [review]
LGTM

**Review by sindrig — APPROVED** [review]
LGTM (only reviewed devops+core files, not the business logic)

File: libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts [hunk 1]
```diff
@@ -1,38 +1,75 @@
...
+import { parseDrivingLicensePayload } from './drivingLicenseMappers'
+import {
+  CONFIG_PROVIDER,
+  GenericLicenseClient,
+  GenericLicenseUserdataExternal,
+  GenericUserLicenseStatus,
+} from '../../licenceService.type'
+import { Config } from '../../licenseService.module'
+
+// TODO(osk) refactor
```
Comment (osk) [inline libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts:24]: Working on this

File: libs/api/domains/license-service/src/lib/licenseService.service.ts [hunk 2]
```diff
@@ -1,71 +1,235 @@
...
+
+    if (ttl > 0) {
+      const cachedData = await this.cacheManager.get(cacheKey)
+
+      if (cachedData) {
+        try {
+          const data = JSON.parse(cachedData as string) as GenericLicenseCached
+
+          // TODO(osk) check again
+          const x = add(data.fetch.updated, { seconds: ttl })
```
Comment (osk) [inline libs/api/domains/license-service/src/lib/licenseService.service.ts:62]: working on this

**Review by thordurhhh — APPROVED** [review]
Looking good. 👍

**Pull request merged**